.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+indent_style = space
+charset = utf-8
+
+[*.{csproj,yml,yaml,conf}]
```

Comment:
With coding standard I think we can remove most of those (at least .cs files I think)